gnovm/pkg/gnolang/values.go

```diff
@@ -890,6 +899,33 @@ type TypedValue struct {
 	N [8]byte `json:",omitempty"` // numeric bytes
 }
 
+// magic 8 bytes to denote a readonly wrapped non-nil V of mutable type that is
```

Comment:
of any type, bad comment. I think it also gets set for StringValue, etc, which should be ok